### PR TITLE
[MI-143] Add targets resource

### PR DESCRIPTION
**Pull request by joseconsador**
Adding the targets resource. No test was written - methods are covered in the BasicTest class.

/cc @miogalang @jwswj @mmolina @atroche 

### References
 - Jira link: https://zendesk.atlassian.net/browse/MI-143

### Risks
 - Medium - the endpoint might change

### CODE REVIEW

File: tests/Zendesk/API/UnitTests/TargetsTest.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Zendesk\API\UnitTests;
+
+class TargetsTest extends BasicTest
+{
+    /**
+     *  This function is a placeholder for this whole unit tests until other cases are introduced that require testing.
+     */
+    public function testPlaceholder()
```
Comment (miogalang) [inline tests/Zendesk/API/UnitTests/TargetsTest.php]: If no tests are needed then just don't create the file I guess

**Comment by joseconsador** [comment]
:haircut:

**Comment by miogalang** [comment]
:+1: Please make the travis test pass before merging

**Comment by jwswj** [comment]
:+1:

**Pull request merged**